src/sdk/namespace_blob.js

```diff
@@ -46,12 +46,9 @@ class NamespaceBlob {
         this.connection_string = connection_string;
         this.container = container;
         this.blob = azure_storage.BlobServiceClient.fromConnectionString(connection_string);
-        // needed only for generateBlockIdPrefix() and get_block_id() functions
-        // TODO: replace old lib functions with new impl of block_id getters
-        this.old_blob = azure_storage.get_old_blob_service_conn_string(connection_string);
         this.account_name = account_name;
         this.account_key = account_key;
-        this.container_client = azure_storage.get_container_client(this.blob, this.container);
+        this.container_client = this.blob.getContainerClient(this.container);
```

Comment:
I feel it's a bit confusing for developers to initiate the container client directly (getContainerClient() ) but the blob client through the wrapper (azure_storage.get_blob_client())  
WDYT?